config/schema.sql

```diff
@@ -103,6 +103,9 @@ CREATE TABLE oauth_data (
     code text NOT NULL
 );
 
+CREATE UNIQUE INDEX on oauth_data (code);
+ALTER TABLE oauth_data ADD CONSTRAINT unique_code UNIQUE (code);
```

Comment:
I think we should go with the `CREATE UNIQUE INDEX` -- it seems more obviously like what we want. It's also unclear to me that a `UNIQUE` constraint is the same thing as a `UNIQUE INDEX` (I'm pretty sure they are in MySQL, but I'm not sure about Postgres). After all, a `UNIQUE` constraint doesn't need to speed-up querying, which probably opens up the design space a little (I'm imagining a hash table w/ no range queries for the constraint vs a  b-tree for the index).